public/global.css

```
@@ -46,3 +46,21 @@ a {
 .lock-screen {
   cursor: wait;
 }
+
+.transition {
+  transition: transform 0.2s ease-in-out;
+}
+
+.transition:hover,
+.transition:checked:hover {
```

It’s enough to use one selector
```suggestion
.transition:hover {
```